src/ast/nodes/ForOfStatement.ts

```diff
@@ -18,14 +19,20 @@ import type { PatternNode } from './shared/Pattern';
 import { includeLoopBody } from './shared/loops';
 
 export default class ForOfStatement extends StatementBase {
-	declare await: boolean;
 	declare body: StatementNode;
 	declare left: VariableDeclaration | PatternNode | MemberExpression;
 	declare right: ExpressionNode;
 	declare type: NodeType.tForOfStatement;
 
-	createScope(parentScope: Scope): void {
-		this.scope = new BlockScope(parentScope);
+	get await(): boolean {
```

Comment:
We don't actually read from this boolean ever, it gets assigned as part of parseNode from the value on the esTreeNode. We could technically just remove the getter and make it a set only property to make the coverage check happy, but that seems a bit strange.

Reply:
I'm ok either way. For me it is only important that we have the setter to remind us that the AST property exists. Then we can add the getter once we actually find out we need the property for something.

Reply:
I'll keep the getter.